api/v1alpha1/openstack_ansibleee_types.go

```diff
@@ -31,6 +31,20 @@ const (
 	OpenStackAnsibleEEContainerImage = "quay.io/openstack-k8s-operators/openstack-ansibleee-runner:latest"
 )
 
+const (
```

Comment:
Do we need add multiple constant blocks?

Reply:
This is not requited but I added it just to make the grouping more clear.